SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -438,6 +438,49 @@ \subsection{Advanced Module Configuration}
 displayed (Figure \ref{fig:ConfigITKDefault}); these messages are sorted in
 alphabetical order by module names.
 
+Those who prefer to build ITK using the command line are referred to the online
+cmake command-line tool
+documentation\footnote{\url{https://cmake.org/cmake/help/latest/manual/cmake.1.html}.
+Only some typical use cases are shown here for reference.
+
+\begin{itemize}
+\item \textbf{Example 1}: Build all default modules.
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+cmake [-DITK_BUILD_DEFAULT_MODULES:BOOL=ON]
+      /ITK-build
+\end{minted}
+\normalsize
+
+The default is \code{ITK\_BUILD\_DEFULT\_MODULES=ON}, so it is optional to
+have the \code{-D} argument.
+
+\item \textbf{Example 2}: Enable specific group(s) of modules.
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+cmake -DITKv_BUILD_DEFAULT_MODULES:BOOL=OFF
+      -DBUILD_EXAMPLES:BOOL=OFF
+      -DITKGroup_{Group1}:BOOL=ON
+      [-DITKGroup_{Group2}:BOOL=ON]
+      /ITK-build
+\end{minted}
+\normalsize
+
+\code{BUILD\_EXAMPLES=ON} requires \code{ITK\_BUILD\_DEFAULT\_MODULES=ON},
+therefore it is necessary to add \code{-DBUILD\_EXAMPLES=OFF} to the command
+line.
+
+\item \textbf{Example 3}: Enable specific modules.
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+cmake -DITK_BUILD_DEFAULT_MODULES:BOOL=OFF
+      -DBUILD_EXAMPLES:BOOL=OFF
+      -DModule_{Module1}:BOOL=ON
```

Comment:
Same as for group, better to use concrete example such as `Module_ITKFEM` for a non-default built-in module and `Module_Cuberille` for a remote module.

Reply:
+1